chromadb/api/models/Collection.py

```diff
@@ -50,9 +51,9 @@ def __init__(
         embedding_function: Optional[EmbeddingFunction] = ef.DefaultEmbeddingFunction(),
         metadata: Optional[CollectionMetadata] = None,
     ):
+        super().__init__(name=name, metadata=metadata, id=id)
```

Comment:
This fixes pydantic v2 because it needs attrs inited before it can assign to PrivateAttrs